src/Components/SignUpPage/SignUpPage.css

```diff
@@ -157,3 +166,57 @@
   padding: 5px;
   margin-left: 150px;
 }
+
+/* Register page styles */
+.close-reg-modal{
+  margin-top: 15px;
+  margin-left: 15px;
+}
+
+.reg-details{
+  padding: 20px;
+  width: 100%;
+  max-width: 70%;
+  margin: auto;
+  height: 80vh;
+}
+
+.reg-details > h1{
+  margin-bottom: 20px;
+}
+
+.reg-details input[type="text"]{
+  margin-bottom: 20px;
+  padding: 15px;
+  width: 100%;
+  max-width: 80%;
+  outline: none;
+}
+
+.reg-details input[type="email"]{
+  margin-bottom: 20px;
+  padding: 15px;
+  width: 100%;
+  max-width: 80%;
+  outline: none;
+}
+
+.reg-details input[type="password"]{
+  margin-bottom: 20px;
+  padding: 15px;
+  width: 100%;
+  max-width: 80%;
+  outline: none;
```

Comment:
Why not shrink all these styles into `.reg-details input`?

Reply:
Done sir